### PR TITLE
flip arrow added for sorting in catagories page

### PR DESCRIPTION
**Pull request by maccrin**
Description:As a user, I want to see what sorting option and direction I chose from the indicator by the sort option itself

Jira task link:https://hackyourfuture-dk.atlassian.net/jira/software/projects/LIST/boards/8?selectedIssue=LIST-47

Screenshots (if applicable):

https://github.com/HackYourFuture-CPH/CINEMANIA/assets/113373210/6a6fae4a-df12-4341-8b3d-f46da14b058c

### CODE REVIEW

File: packages/client/src/components/Select/CustomSelect.jsx [hunk 1]
```diff
@@ -1,63 +1,93 @@
 import React from 'react';
 import { FormControl, InputLabel, Select, MenuItem } from '@mui/material';
 import { useMovieList } from '../../context/movieListContext';
-import styled from '@emotion/styled';
+import FilterListIcon from '@mui/icons-material/FilterList';
 
 const CustomSelect = () => {
-  const { sortBy, onSortMovies, onChangeDirection } = useMovieList();
-
+  const { sortBy, onSortMovies, onChangeDirection, isClickedSame } =
```
Comment (shpomp) [inline packages/client/src/components/Select/CustomSelect.jsx:7]: `isClickedSame` might be a bit misleading... Maybe a very explicit `isSameOptionClicked` would be better?

**Review by shpomp — CHANGES_REQUESTED** [review]
Very nice! 
Just small naming change 🙏🏽

File: packages/client/src/components/Select/CustomSelect.jsx (hunk 1 above)
Comment (maccrin) [reply packages/client/src/components/Select/CustomSelect.jsx:7]: Thanks for review, will fix it.

**Review by shpomp — APPROVED** [review]
👏🏽

**Pull request merged**